pkg/stanza/fileconsumer/config.go

```diff
@@ -172,6 +173,25 @@ func (c Config) Build(set component.TelemetrySettings, emit emit.Callback, opts
 		t = tracker.NewFileTracker(set, c.MaxConcurrentFiles/2)
 	}
 	set.Logger = set.Logger.With(zap.String("component", "fileconsumer"))
+
+	meter := set.MeterProvider.Meter("otelcol/fileconsumer")
+
+	openedFiles, err := meter.Int64UpDownCounter(
+		"fileconsumer"+"/"+"open_files",
```

Comment:
Was this broken up because there was an intention to use constants?

Reply:
Not really, but good point. I will make those constants!